web/modules/custom/sfgov_vaccine/js/filter_sites.js

```diff
@@ -1,329 +1,304 @@
-            hideCount();
-            hideSites();
-            showNoResultsMessage();
+            hideCount()
+            hideSites()
+            showNoResultsMessage()
           } else {
             // If "Only show sites with available appointments" is not checked and
             // there are sites that meet the selected criteria.
-            hideNoResultsMessage();
-            showSites();
-            showCount();
+            hideNoResultsMessage()
+            showSites()
+            showCount()
           }
         }
       }
 
       // Responsive layout.
-      function layoutChange(e) {
+      function layoutChange (e) {
         if (e.matches) {
-          $(".vaccine-filter__filters").appendTo(".group--right");
+          $('.vaccine-filter__filters').appendTo('.group--right')
         } else {
-          $(".vaccine-filter__filters").appendTo(
-            ".vaccine-filter__filter-top > div"
-          );
+          $('.vaccine-filter__filters').appendTo(
+            '.vaccine-filter__filter-top > div'
+          )
         }
       }
 
       // Scroll to Top.
-      function scrollUp(speed) {
-        let newPosition = sectionCount.offset().top - 150;
-        $("html, body").animate({ scrollTop: newPosition }, speed);
+      function scrollUp (speed) {
+        const newPosition = sectionCount.offset().top - 150
+        $('html, body').animate({ scrollTop: newPosition }, speed)
+      }
+
+      function getSiteData (el) {
+        return el.hasAttribute('data-site')
+          ? tryParse(el.getAttribute('data-site')) || {}
+          : {}
+      }
+
+      function tryParse (str) {
+        try {
+          return JSON.parse(str)
+        } catch (error) {
+          return undefined
+        }
+      }
+
+      function formatDistance (distance) {
+        return isNaN(distance)
+          ? ''
+          : Math.round(distance * 10) / 10 + 'mi'
+      }
+
+      // @see https://en.wikipedia.org/wiki/Haversine_formula
+      // @see https://simplemaps.com/resources/location-distance
+      function getDistance ({ lat: lat1, lng: lng1 }, { lat: lat2, lng: lng2 }) {
+        lat1 = deg2rad(lat1)
+        lat2 = deg2rad(lat2)
+        const latOff = lat2 - lat1
+        const lngOff = deg2rad(lng2 - lng1)
+        const a = (
+          Math.pow(Math.sin(latOff / 2), 2) +
+          Math.cos(lat1) * Math.cos(lat2) * Math.pow(Math.sin(lngOff / 2), 2)
+        )
+        const d = 2 * R * Math.asin(Math.sqrt(a))
+        return Math.round(d * 10) / 10 // round to one decimal point
+      }
+
+      function deg2rad (deg) {
+        return deg * Math.PI / 180
+      }
+
+      function sortAscending (a, b) {
+        return a < b ? -1 : a === b ? 0 : 1
+      }
+
+      function rangesOverlap ([x1, x2], [y1, y2]) {
+        return x1 <= y2 && y1 <= x2
+      }
+
+      function inCityBounds ({ lat, lng }) {
+        return (
+          lng < -122.93 || lng > -121.54 ||
+          lat < 37.0000 || lat > 38.0200
```

Comment:
```suggestion
          (lng > -122.93 || lng < -121.54) &&
          (lat > 37.0000 || lat < 38.0200)
```
previous comparison was checking for out of city bounds.  since we're now comparing in city bounds, think the comparison operators have to flip